flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/flink/SQLJobSubmission.java

```diff
@@ -90,6 +91,11 @@ public SQLJobSubmissionBuilder addJar(Path jarFile) {
 			return this;
 		}
 
+		public SQLJobSubmissionBuilder addJars(List<Path> jarFiles) {
+			this.jars.addAll(jarFiles.stream().map(path -> path.toAbsolutePath().toString()).collect(Collectors.toList()));
```

Comment:
```suggestion
			jarFiles.forEach(this::addJar);
```

Reply:
Ah, sweet! Thanks! You see I haven't worked much with Java 8 :)